src/main/java/com/modrinth/minotaur/ModrinthExtension.java

```diff
@@ -34,10 +36,11 @@ public ModrinthExtension(Project project) {
         versionType = project.getObjects().property(String.class).convention("release");
         gameVersions = project.getObjects().listProperty(String.class).empty();
         loaders = project.getObjects().listProperty(String.class).empty();
-        dependencies = project.getObjects().listProperty(Dependency.class).empty();
+        dependencies = project.getObjects().newInstance(DependenciesConfiguation.class, project.getObjects());
```

Comment:
Is there any way this could be added while still keeping backwards compatibility? This would need to get a version bump to the 3.x series (since the current recommended version is `2.+` with auto updating), and I don't want to do that quite yet.

Reply:
I thought of this after making the PR, If your fine with keeping this open I can work on making this backwards compatible

Reply:
Backwards compatibility added, either method can be used now, or both. tested using this in local project
```gradle
	dependencies = [
		new ModDependency("tsi4hUoN", "optional"),
		new VersionDependency("igy4JZo5", DependencyType.INCOMPATIBLE)
	]
	
	dependencies {
		incompatible.mod("incompatible-mod-dependency")
		incompatible.version("incompatible-version-dependency")
		
		optional.mod("optional-mod-dependency")
		optional.version("optional-version-dependency")
		
		required.mod("required-mod-dependency")
		required.version("required-version-dependency")
	}
	```